### PR TITLE
Need go back to project root before run testing

### PR DESCRIPTION
**Pull request by fffx**
Since the previous step is:
```
$ cd android
$ ./gradlew bundleRelease
```
so next step we need to go back to the project root folder

### CODE REVIEW

**Comment by fffx** [comment]
@Simek @rachelnabors  Thanks, it's updated

File: docs/signed-apk-android.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ _Note: In order for Google Play to accept AAB format the App Signing by Google P
 
 ### Testing the release build of your app
 
-Before uploading the release build to the Play Store, make sure you test it thoroughly. First uninstall any previous version of the app you already have installed. Install it on the device using:
+Before uploading the release build to the Play Store, make sure you test it thoroughly. First uninstall any previous version of the app you already have installed. Install it on the device using(in project root):
```
Comment (Simek) [inline docs/signed-apk-android.md:96]: Space is missing between last word and round bracket.

What do you think about putting this information into the sentence? 
* "Install it on the device using the following command in the project root:"

Comment (fffx) [reply docs/signed-apk-android.md]: Any language advice is accepted, as I am not an English speaker

Comment (Simek) [reply docs/signed-apk-android.md]: I'm not the one too, thank you for updating this PR during review!

**Pull request merged**